### PR TITLE
Added Warp Terminal Theme

### PR DESCRIPTION
**Pull request by jayzone91**
A ported Theme of Cyberdream for the Warp terminal.

### CODE REVIEW

File: extras/warp/cyberdream.yml
Comment (scottmckendry) [inline extras/warp/cyberdream.yml:1]: Great stuff @jayzone91 😁 

Just need to add this as a Lua template so we can generate a light theme alongside the default one.

Check [the contributing guide](https://github.com/scottmckendry/cyberdream.nvim/blob/main/CONTRIBUTING.md#adding-extras-) for instructions. Let me know if you have any questions!

**Review by scottmckendry — CHANGES_REQUESTED** [review]

File: extras/warp/cyberdream.yml
Comment (jayzone91) [reply extras/warp/cyberdream.yml:1]: I Added the template as Lua 😀

**Comment by scottmckendry** [comment]
Hmm, looks like a couple of CI jobs are failing... You may want to enable actions in your fork, that way the auto-commit action can write the extras back to the repo.

Alternatively you can run the same thing locally with the following (from the root of the project):

```sh
nvim -u NONE -E -R --headless --cmd "set rtp^=." --cmd "lua require('cyberdream.extra').generate_all_extras()" --cmd qa
```

**Review by scottmckendry — COMMENTED** [review]
All looks good to me! Just need to fix the indentation on the two lua files and we should be good to merge (once all CI jobs are passing 😉)

**Comment by jayzone91** [comment]
`nvim -u NONE -E -R --headless --cmd "set rtp^=." --cmd "lua require('cyberdream.extra').generate_all_extras()" --cmd qa`
runs without problems. I've forgotten to format the file with stylua.

**Review by scottmckendry — APPROVED** [review]

**Comment by scottmckendry** [comment]
Thanks for the PR @jayzone91!

**Pull request merged**